Create comprehensive GitHub Copilot instructions for .NET Aspire development workflow (#7)
webreidi/codel-cloud-native: .github/copilot-instructions.md

# Codel Cloud Native - GitHub Copilot Instructions

**Codel Cloud Native** is a .NET 9.0 Aspire application featuring a Blazor WebAssembly frontend that implements "Codele" - a developer-themed Wordle game. The application uses SQL Server and Redis containers orchestrated through .NET Aspire's Developer Control Plane.

**Always reference these instructions first and fallback to search or bash commands only when you encounter unexpected information that does not match the info here.**

## Working Effectively

### Prerequisites - CRITICAL SETUP REQUIRED
**You MUST install .NET 9.0 SDK and Aspire workload before any development work:**

1. **Install .NET 9.0 SDK** (takes ~3 minutes, NEVER CANCEL):
   ```bash
   curl -sSL https://dotnet.microsoft.com/download/dotnet/scripts/v1/dotnet-install.sh | bash -s -- --channel 9.0 --install-dir ~/.dotnet
   export PATH=$HOME/.dotnet:$PATH
   ```
   
2. **Verify .NET 9.0 installation**:
   ```bash
   dotnet --version  # Should show 9.0.x
   ```

> 3. **Install Aspire workload** (takes ~2-3 minutes, NEVER CANCEL):

Reviewer: Aspire workload is no longer needed for .NET9+. Remove this command.

> 3. **Install Aspire workload** (takes ~2-3 minutes, NEVER CANCEL):

Reviewer: ```suggestion
**Install Aspire templates**
```

   ```bash

>    dotnet workload install aspire

Reviewer: ```suggestion
dotnet new install Aspire.ProjectTemplates::9.4.0 --force
```

   ```

4. **Verify Aspire installation**:
   ```bash

>    dotnet workload list  # Should show 'aspire' in the list

Reviewer: ```suggestion
   dotnet new list aspire # Should show 'aspire' in the list
```
